openapi/schemas/processes-dru/cwl.yaml

```diff
@@ -0,0 +1,1650 @@
+    properties:
+      processDescription:
+        $ref: "https://raw.githubusercontent.com/opengeospatial/ogcapi-processes/master/openapi/schemas/processes-core/process.yaml"
+      immediateDeployment:
+        type: boolean
+        title: immediateDeployment
+        default: true
+      deploymentProfileName:
+        type: string
+        format: uri
+        enum:
+        # if the CWL has a DockerRequirement and is a 'class: CommandLineTool'
+        - "http://www.opengis.net/profiles/eoc/dockerizedApplication"
+        # when the CWL is a 'class: Workflow'
+        - "http://www.opengis.net/profiles/eoc/workflow"
+      executionUnit:
+        oneOf:
+          - $ref: "#/$definitions/CWLExecutionUnit"
+          - type: array
+            items: "#/$definitions/CWLExecutionUnit"
+    required:
+      - executionUnit
+      - processDescription
+    additionalProperties: false
+  CWLExecutionUnit:
+    description: CWL definition use to deploy a process.
+    type: object
+    properties:
+      unit:
+        $ref: '#/$definitions/CWL'
+  CWL:
+    oneOf:
+      - $ref: '#/$definitions/CWLAtomic'
+      - $ref: '#/$definitions/CWLGraph'
+  CWLKeywordList:
+    type: array
+    title: KeywordList
+    description: Keywords applied to the process for search and categorization purposes.
+    default: []
+    items:
+      type: string
+      title: keyword
+      minLength: 1
+  CWLIdentifier:
+    anyOf:
+    - type: string
+      title: UUID
+      description: Unique identifier.
+      format: uuid
+      pattern: ^[a-f0-9]{8}(?:-?[a-f0-9]{4}){3}-?[a-f0-9]{12}$
+    - type: string
+      title: ID
+      description: |
+        Generic identifier. This is a user-friendly slug-name.
+        This will represent the process to be deployed.
+      pattern: ^[A-Za-z0-9]+(?:[-_][A-Za-z0-9]+)*$
+    title: CWLIdentifier
+    description: Reference to the process identifier. If CWL is provided within
+      a process deployment payload, this can be omitted. If used in a deployment
+      with only CWL details, this information is required.
+  CWLIntent:
+    type: array
+    title: CWLIntent
+    items:
+      type: string
+      title: item
+      description: |
+        Identifier URL to a concept for the type of computational operation accomplished by this process
+        (see example operations: http://edamontology.org/operation_0004).
+      format: url
+      pattern: '^((?:http|ftp)s?://)?(?!.*//.*$)(?:(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}\.?)|localhost|\[[a-f0-9:]+\]|\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})(?::\d+)?(?:/?|[/?]\S+)$'
+  CWLRequirements:
+    oneOf:
+    - $ref: '#/$definitions/CWLRequirementsMap'
+    - $ref: '#/$definitions/CWLRequirementsList'
+    title: CWLRequirements
+    description: Explicit requirement to execute the application package.
+  CWLRequirementsMap:
+    anyOf:
+    - $ref: '#/$definitions/DockerRequirementMap'
+    - $ref: '#/$definitions/DockerGpuRequirementMap'
```

Comment:
Oh, are people still using `DockerGpuRequirement` instead of `CUDARequirement`?

Reply:
I thought that I had removed it. Let me double check because CRIM also uses and support `CUDARequirement`.

Reply:
Done. Forgot to remove that reference but the definition was already removed.